### ISSUE TITLE
Issues with True_positive, False_positive, False_negative functions in pyG 2.3

### ISSUE THREAD

**Issue by balexx2** [pyg-team/pytorch_geometric#7692]
### 🐛 Describe the bug

The error message indicates that the 'torch_geometric.utils' module does not have the attribute or method 'true_negative'.
Here is the line of code where this error is orginating:

**TN = torch_geometric.utils.true_negative(mincaps[~no_step_mask], actual_mincaps[~no_step_mask], 2).sum()**

This line of code is trying to call a function true_negative from the torch_geometric.utils module, but the error message is stating that this function does not exist, hence the AttributeError. the line is extracted from here
**argmins = mins.argmax(dim=1) # yes, mins are logits for the min position, hence argmax
        mincaps = attrs[torch.arange(batch.num_graphs), argmins, 1]
        TN = torch_geometric.utils.true_negative(mincaps[~no_step_mask], actual_mincaps[~no_step_mask], 2).sum()
        FP = torch_geometric.utils.false_positive(mincaps[~no_step_mask], actual_mincaps[~no_step_mask], 2).sum()
        self.mincaps_FP += FP
        self.mincaps_TN += TN
        return mincaps**

### Environment

* PyG version:2.3.0
* PyTorch version: 2.0.1
* OS: Ubuntu 18.04
* Python version: 3.10
* CUDA/cuDNN version: CPU
* How you installed PyTorch and PyG (`conda`, `pip`, source): conda
* Any other relevant information (*e.g.*, version of `torch-scatter`):

**Comment by rusty1s** [comment]
We removed our metrics package a while ago. You can use `torchmetrics` as a replacement, our leverage older PyG versions/copy paste the code from here: https://github.com/pyg-team/pytorch_geometric/blob/1.7.2/torch_geometric/utils/metric.py
